doc/contributing/issues.md

```diff
@@ -72,5 +73,12 @@ When triagging issues and PRs:
   dismissive from the point of view of the reporter/author.
   Always try to communicate the reason for closing the issue/PR.
 
+## Blocking Issues
+
+Collaborators have the ability to block issues from getting implemented and landed into
```

Comment:
The wording here is problematic. We cannot block anything from being implemented. We block PRs from being merged. None of us have the right or ability to block anyone from implementing anything.